### ISSUE TITLE
[Enhancement] Allow heading level styling in section header

### ISSUE THREAD

**Issue by RasmusKjeldgaard** [kirbydesign/designsystem#4444]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
I'd like to be able to adjust the visual appearance of the heading inside a `kirby-section-header`, so it matches the Figma property, which is:
- normal (maps to .kirby-text-normal and h4)
- medium (maps to .kirby-text-medium and h3)
- large (maps to .kirby-text-large and h2)

## Describe the solution you'd like
It should default to medium, and allow consumers to override by using either a heading element or class, as mapped out above.

This should be documented with an example, in the section-header showcase and added to the API table if applicable.

## Have you considered any alternatives?
No, this is the perfect solution that @henrikvoetmand carefully thought up and we need it right now (Friday vibes 🌞)

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [x] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [x] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [x] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [x] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".
